third_party/ibis/ibis_impala/api.py

```diff
@@ -63,7 +65,9 @@ def impala_connect(
         user=user,
         password=password,
         pool_size=pool_size,
-        hdfs_client=hdfs_client
+        hdfs_client=hdfs_client,
```

Comment:
I don't know if this will work since the [connect() method ](https://github.com/ibis-project/ibis/blob/1.4.0/ibis/backends/impala/__init__.py#L41) doesn't support the new flags. We'll probably have to write our own connect() signature and copy the original connect logic.